### PR TITLE
Doc: Add snmp and snmptrap migration guides

### PR DESCRIPTION
**Pull request by edmocosta**
Closes: https://github.com/logstash-plugins/logstash-integration-snmp/issues/11

### CODE REVIEW

**Review by karenzone — COMMENTED** [review]
From my local build: 
`WARNING: ../../logstash-docs/docs/plugins/inputs/snmp.asciidoc: line 403: id assigned to section already in use: plugins-inputs-snmp-use_provided_mibs`

The section and id for `use_provided_mibs` is duplicated starting at lines 337 and 403. 
If you want this option to appear in both places, you'll need to give the second occurrence a different id.

**Comment by karenzone** [comment]
As you know, we're not generating these files yet.  I'm testing locally and I also mocked up this PR as a testbed: https://github.com/elastic/logstash-docs/pull/1629.  I can explain in more detail if you're interested.

**Comment by edmocosta** [comment]
> From my local build: `WARNING: ../../logstash-docs/docs/plugins/inputs/snmp.asciidoc: line 403: id assigned to section already in use: plugins-inputs-snmp-use_provided_mibs`
> 
> The section and id for `use_provided_mibs` is duplicated starting at lines 337 and 403. If you want this option to appear in both places, you'll need to give the second occurrence a different id.
> 
> I'll do a more thorough review after we clear the docs build issues.

Hey @karenzone, thanks for reviewing! It was indeed duplicated.

File: docs/input-snmp.asciidoc [hunk 1]
```diff
@@ -494,4 +485,19 @@ input {
 [id="plugins-{type}s-{plugin}-common-options"]
 include::{include_path}/{type}.asciidoc[]
 
+[id="plugins-{type}s-{plugin}-migration-guide-input-snmp"]
+==== Migrating from logstash-input-snmp
+
+The `logstash-integration-snmp` `snmp` input plugin does support the same https://github.com/logstash-plugins/logstash-input-snmp[logstash-input-snmp] functionalities. There are a few changes that might
```
Comment (karenzone) [inline docs/input-snmp.asciidoc:491]:
```suggestion
The `logstash-integration-snmp` `snmp` input plugin does support the same `logstash-input-snmp` functionalities. There are a few changes that might
```

**Comment by karenzone** [comment]
@edmocosta, I'm working on structure and asciidoctor format for the idea that we discussed. I'll have something for you to look at tomrrow.

**Pull request closed without merging**